### PR TITLE
expose the Pose structure to JS

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

File: libraries/controllers/src/controllers/Pose.h [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #ifndef hifi_controllers_Pose_h
 #define hifi_controllers_Pose_h
 
+#include <QScriptValue>
```
Comment (jherico) [inline libraries/controllers/src/controllers/Pose.h]: can use forward declaration here.  Also, `<QtScript/QScriptValue>`

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3eedfd369e7c02866d8d76e6228c7ee8aac7afed available at: https://gist.github.com/2bd58e5d43f35e99099e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6140.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6140.dmg

**Comment by ZappoMan** [comment]
damn it.. wth did I merge this line of code... `#include "../../entities/src/EntityItemID.h"`

**Comment by hifi-gustavo** [comment]
Build console output for commit: ffd277d4d5d75a1a59fea8f6505b3449c9b00aad available at: https://gist.github.com/2bd58e5d43f35e99099e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6140.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6140.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6bd1e593059cf45740cb4dadab4681c57744fe95 available at: https://gist.github.com/2bd58e5d43f35e99099e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6140.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6140.dmg

**Pull request merged**